bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -362,7 +362,7 @@ spec:
 
           hybrid_overlay_flags=
           if [[ -n "{{.OVNHybridOverlayEnable }}" ]]; then
-            hybrid_overlay_flags="--enable-hybrid-overlay"
+            hybrid_overlay_flags="--enable-hybrid-overlay --ignore-nodes=\"kubernetes.io/os=windows\""
```

Comment:
@JacobTanenbaum don't you want `--no-hostsubnet-nodes` ?

Reply:
you are right sorry. Fixed